cdap-app-fabric/src/main/java/co/cask/cdap/app/runtime/RunIds.java

```diff
@@ -77,12 +78,13 @@ public static RunId generate(long timeMillis) {
   }
 
   /**
-   * @return time in milliseconds from the UUID if it is a time-based UUID, -1 otherwise.
+   * @return time from the UUID if it is a time-based UUID, -1 otherwise.
    */
-  public static long getTimeMillis(RunId runId) {
```

Comment:
ohh ok. Will add it. But not sure who will call `getTimeMillis`, as all calls to this method are for setting the store and I thought they will use the `getTime`.   
`AppMetadataStore` was also calling `getTimeMillis` but it was converting the returned value to the seconds.